locust_tasks/locustfile.py

```diff
@@ -142,7 +165,7 @@ def start_page(self):
     @task(2)
```

Comment:
You can't have a number of the @task.

Suggest search the file for '@task(' to see if there any others.

Reply:
good spot - will do

Reply:
done and pushed